code/utils/xcmp/src/config.rs

```diff
@@ -0,0 +1,91 @@
+use clap::{clap_derive::ArgEnum, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[clap()]
```

Comment:
Is the empty attribute required?

Reply:
you mean if need clap or if need () in clap?

Reply:
If the entire attribute is required - afaik it doesn't do anything

Reply:
removed